backend/core/views.py

```diff
@@ -293,3 +300,52 @@ def competence_reviews_outstanding(self, request, pk=None):
 #             result[team.id]["permissions"].append(permission)
 
 #     return Response(result)
+
+
+class BulkAddUsersToTeamView(LoginRequiredMixin, FormView):
+    form_class = BulkAddUsersToTeamForm
+    template_name = "admin/core/bulk_add_users_form.html"
+    success_url = reverse_lazy("bulk_add_users_to_team")
+
+    def get_login_url(self):
+        return reverse("admin:login")
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        team_id = self.kwargs["team_id"]
+        team = Team.objects.get(id=team_id)
+        context["team"] = team
+        return context
+
+    def form_valid(self, form):
+        email_addresses = form.cleaned_data["email_addresses"]
+        team_id = self.kwargs["team_id"]
+        team = Team.objects.get(id=team_id)
+        users_added_to_team = self.add_users_to_team(team, email_addresses)
+        if users_added_to_team:
+            messages.success(
+                self.request,
+                f"The following users were successfully added to the \"{team}\" team: {', '.join(users_added_to_team)}",
+            )
+            return redirect(
+                reverse("admin:core_team_change", kwargs={"object_id": team.id})
+            )
+        else:
+            messages.error(
+                self.request,
+                f'No users were added to the "{team.name}" team. Make sure the users and/or entered email addresses exist and try again.',
+            )
+            return redirect(
+                reverse("bulk_add_users_to_team", kwargs={"team_id": team.id})
+            )
+
+    def add_users_to_team(self, team, email_addresses):
+        if team:
+            users = User.objects.filter(email__in=email_addresses)
```

Comment:
What if there are 20 email addresses entered and one of them is invalid? Should the admin user need to read through the list of email addresses and play spot the difference? 

Proper form validation is important. There should be an error message that says which emaild aren't found.